spec/views/session/index.html.erb_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'session/index', type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
@cdmo  shouldn't the directory be `spec/views/sessions/`?

Reply:
:nods: